### DISCUSSION TITLE
Search only viral reads

### DISCUSSION THREAD

**Discussion by Ysonia** [bbuchfink/diamond#606] [Q&A]
Hi All, 
I am doing virome analysis of chicken fecal samples. I am using the following command to get only viral reads but when i check accession id from the output files , the reads belongs to different bacteria. please guide me 

diamond makedb --in nr.gz --db nr --taxonmap prot.accession2taxid2.gz --taxonnodes nodes.dmp -p 64 -t tmp_dir
diamond blastx --query transcripts.fasta --out viral_matches.txt --db nr --taxonlist 10239 --evalue 1e-3 -p 64 -t tmp_dir

Also if there is some solution as i am trying to learn.

**Comment by bbuchfink** [comment]
Could you please provide an example output for a read that maps incorrectly to bacteria?
